pararead/logs.py

```diff
@@ -332,7 +332,7 @@ def _level_from_verbosity(verbosity):
     elif isinstance(verbosity, int):
         # Allow negative value to mute even ERROR level but not CRITICAL.
         # Also handle excessively high verbosity request.
-        v = min(min(verbosity, 0), len(LEVEL_BY_VERBOSITY) - 1)
+        v = min(max(verbosity, 0), len(LEVEL_BY_VERBOSITY) - 1)
```

Comment:
This change invalidates the comments above it.

Reply:
go ahead and change...

Reply:
I don't know what the change is trying to do, though; what's the intended behavior?

Reply:
Is it that we no longer want the comments to be true, or that they should be?

Reply:
I don't think I made that change, or at least I don't recall it. TO be honest I have no idea what's going on with this logger stuff, still, or what these changes would do... It's not immediately clear to me and I would need some time to try to figure out how your logging system works... I just use it (and it's awesome)